loopback/loopback.go

```diff
@@ -0,0 +1,539 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+// +build darwin
```

Comment:
any reason to limit this to darwin? should work on linux, too right? Maybe build for non-windows?